po/fr.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/info.rs:76 src/info.rs:128
+msgid "Keywords"
+msgstr "Mots clés"
+
+#: src/info.rs:77 src/info.rs:130
+msgid "Snapshot URL"
+msgstr "URL de l'instantané"
+
+#: src/info.rs:93
+msgid "No"
+msgstr "Non"
+
+#: src/info.rs:113
+msgid "Optional Deps"
+msgstr "Dépendances optionnelles"
+
+#: src/install.rs:131 src/install.rs:545 src/install.rs:944
+msgid "no architecture"
+msgstr "pas d'architecture"
+
+#: src/install.rs:138
+msgid "can't build package as root"
+msgstr "impossible de compiler le paquet en tant que root"
+
+#: src/install.rs:142
+msgid "failed to parse srcinfo generated by makepkg"
+msgstr "erreur lors de l'analyse du srcinfo généré par makepkg"
+
+#: src/install.rs:171 src/install.rs:442
+msgid "Resolving dependencies..."
+msgstr "Résolution des dépendances..."
+
+#: src/install.rs:220 src/install.rs:225
+msgid "failed to download sources"
+msgstr "échec du téléchargement des sources"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "échec de compilation du paquet"
+
+#: src/install.rs:233
+msgid "parsing pkg list..."
+msgstr "analyse de la liste des paquets..."
+
+#: src/install.rs:261 src/install.rs:1462
+msgid "{}-{} is up to date -- skipping build"
+msgstr "{}-{} est à jour -- inutile de recompiler le paquet"
+
+#: src/install.rs:333 src/query.rs:50 src/lib.rs:84
+msgid "error:"
+msgstr "erreur :"
+
+#: src/install.rs:334
+msgid "could not get news"
+msgstr "impossible de récupérer les nouvelles"
+
+#: src/install.rs:339 src/install.rs:521 src/install.rs:716 src/install.rs:848
+msgid "Proceed with installation?"
+msgstr "Procéder à l'installation ?"
+
+#: src/install.rs:356
+msgid "no targets specified (use -h for help)"
+msgstr "aucune cible spécifiée (utiliser -h pour l'aide)"
+
+#: src/install.rs:434 src/install.rs:461 src/search.rs:324 src/search.rs:360
+#: src/lib.rs:227
+msgid " there is nothing to do"
+msgstr " il n'y a rien à faire"
+
+#: src/install.rs:488
+msgid "can't install AUR package as root"
+msgstr "impossible d'installer le paquet de l'AUR en tant que root"
+
+#: src/install.rs:507
+msgid "Remove make dependencies after install?"
+msgstr "Désinstaller les dépendances de compilation après l'installation ?"
+
+#: src/install.rs:518
+msgid "Proceed to review?"
+msgstr "Procéder à la relecture ?"
+
+#: src/install.rs:559
+msgid "The following packages are not compatible with your architecture:"
+msgstr "Les paquets suivants ne sont pas compatibles avec votre architecture :"
+
+#: src/install.rs:573
+msgid "Would you like to try build them anyway?"
+msgstr "Voulez-vous quand même essayer de les compiler ?"
+
+#: src/install.rs:662
+msgid "packages failed to build: {}"
+msgstr "la compilation des paquets suivants a échouée : {}"
+
+#: src/install.rs:698
+msgid "could not find .SRINFO for '{}'"
+msgstr "impossible de trouver le .SRCINFO pour '{}'"
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "échec de la lecture du dossier : {}"
```

Comment:
is `foo : bar` intentional?

Reply:
The space before :, yes

Reply:
Then you're missing it in other places unless there#s some context i'm unaware of.

Reply:
I checked multiple times and didn't find a place that lacks a space before a colon

Reply:
Ah may have looked at the English by mistake.

Reply:
> is `foo : bar` intentional?

Yes, spacing rules and conventions are different in French when it comes to punctuation.